packages/tokens/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./src/index";
```

Comment:
Should we have the index from the dist folder itself? It shouldn't be dependent on development folders

Reply:
Adding the build system in place in the same PR which will take care of generating the dist folder